### PR TITLE
[1.3.6-prepare][PGSQL] Delete the stopped workflow without deleting t…

### PR DESCRIPTION
**Pull request by CalvinKirs**
…he correspon…

pr    #5066
issue #5048

### CODE REVIEW

**Comment by chengshiwen** [comment]
@CalvinKirs Please resolve the conflicts

**Review by chengshiwen — APPROVED** [review]
+1

File: sql/upgrade/1.3.6_schema/postgresql/dolphinscheduler_ddl.sql [hunk 1]
```diff
@@ -35,3 +35,8 @@ d//
 delimiter ;
 SELECT uc_dolphin_T_t_ds_worker_group_A_ip_list();
 DROP FUNCTION IF EXISTS uc_dolphin_T_t_ds_worker_group_A_ip_list();
+delimiter ;
+
+-- Add foreign key constraints for t_ds_task_instance --
+ALTER TABLE t_ds_task_instance ADD CONSTRAINT foreign_key_instance_id  FOREIGN KEY(process_instance_id) REFERENCES t_ds_process_instance(id) ON DELETE CASCADE;
+
```
Comment (chengshiwen) [inline sql/upgrade/1.3.6_schema/postgresql/dolphinscheduler_ddl.sql:42]: After comparison, there are still some differences for `1.3.6_schema` between `1.3.6-prepare` and `dev`. I want to sync the `1.3.6_schema` with the `dev` branch, so the following adjustments will be better 😃

```
-- Add foreign key constraints for t_ds_task_instance --
delimiter ;
ALTER TABLE t_ds_task_instance ADD CONSTRAINT foreign_key_instance_id  FOREIGN KEY(process_instance_id) REFERENCES t_ds_process_instance(id) ON DELETE CASCADE;
```

**Comment by xingchun-chen** [comment]
+1

**Pull request merged**